parts/k8s/kubernetescustomscript.sh

```diff
@@ -307,34 +329,18 @@ function ensureContainerd() {
 }
 
 function systemctlEnableAndStart() {
-    retrycmd_if_failure 10 1 3 systemctl daemon-reload
-    systemctl enable $1
-    systemctl is-enabled $1
-    enabled=$?
-    for i in {1..900}; do
-        if [ $enabled -ne 0 ]; then
-            systemctl enable $1
-            systemctl is-enabled $1
-            enabled=$?
-        else
-            echo "$1 took $i seconds to be enabled by systemctl"
-            break
-        fi
-        sleep 1
-    done
-    if [ $enabled -ne 0 ]
-    then
-        echo "$1 could not be enabled by systemctl"
-        exit 3
-    fi
-    systemctl_restart 100 1 10 $1
-    retrycmd_if_failure 10 1 3 systemctl status $1 --no-pager -l > /var/log/azure/$1-status.log
-    systemctl is-failed $1
-    if [ $? -eq 0 ]
-    then
+    systemctl_restart 20 1 10 $1
```

Comment:
Simplified this whole function to the following:

- restart systemd service w/ (20) retries
- log the status
- explicit exit if fail
- enable systemd service
- explicit exit if fail